static/js/util/instanceOptions.tsx

```diff
@@ -1,10 +1,10 @@
 export const instanceTypeOptions = [
   {
-    label: "Container",
+    label: "Containers",
```

Comment:
This label is used for the selector on instance creation. There we need the singular form. So maybe pluralize it in place or have two of those?